tutorials/cyber/raw_data_generator/raw_data_generator/raw_data_generator.py

```diff
@@ -0,0 +1,52 @@
+from jinja2 import Environment, PackageLoader, select_autoescape
+import os
+import json
+import logging
+import re
+
+class RawDataGenerator:
+    def __init__(self, hdfs_conn):
+        # Read templates
+        self.env = Environment(loader=PackageLoader('raw_data_generator', 'templates'),)
+        self.schema = ["Time", "EventID", "LogHost", "LogonType", "LogonTypeDescription", "UserName", "DomainName", "LogonID",
+                       "SubjectUserName", "SubjectDomainName", "SubjectLogonID", "Status", "Source", "ServiceName", "Destination",
+                       "AuthenticationPackage", "FailureReason", "ProcessName", "ProcessID", "ParentProcessName", "ParentProcessID", "Raw"]
+        self.hdfs = hdfs_conn
+
+    # Generates raw data from templates
+    def generate_raw_data(self, infilepath, outfilepath, output_format):
+        with self.hdfs.open(infilepath, "rb") as infile:
```

Comment:
it looks like this was intended for your internal dev/testing. Is that correct?

If nto, I'm confused why hdfs is being used here.

Reply:
I think we provided this raw data generator tool as context to share how we generated raw log data, but can also be used as well. The data provided in `/data/wls.tar.gz` is output of this tool and is referenced by the network mapping notebook. Should we still include this code?

Reply:
I have removed requirement for hdfs